framework/wazuh/utils.py

```diff
@@ -140,9 +140,13 @@ def check_sort_fields(allowed_sort_fields, sort_by):
         if type(array[0]) is dict:
             check_sort_fields(set(array[0].keys()), set(sort_by))
 
-            return sorted(array, key=lambda o: tuple(o.get(a) for a in sort_by), reverse=order_desc)
+            return sorted(array,
+                          key=lambda o: tuple(o.get(a).lower() if type(o.get(a)) is str else o.get(a) for a in sort_by),
```

Comment:
In python2 there are both `unicode` and `str` types. Lots of builtin functions return `unicode` objects instead of `str` so I'd be extra careful with this:
```python
lambda o: tuple(o.get(a).lower() if type(o.get(a)) is str or type(o.get(a)) is unicode else o.get(a)
```

In python3 there is no `unicode` type but you can do this:
https://github.com/wazuh/wazuh/blob/eec0738a0d8aa987aaf1346ad7a4226f8e359d37/framework/wazuh/cluster/communication.py#L28-L34